spec/osut_tests_spec.rb

```diff
@@ -4506,6 +4506,72 @@ module M
     sky_area2  = core_skies.sum(&:grossArea)
     expect(sky_area2.round(2)).to eq(0.00)
 
+    # --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- #
+    # SEB case (flat ceiling plenum).
+    file  = File.join(__dir__, "files/osms/in/seb.osm")
+    path  = OpenStudio::Path.new(file)
+    model = translator.loadModel(path)
+    expect(model).to_not be_empty
+    model = model.get
+
+    entry   = model.getSpaceByName("Entry way 1")
+    office  = model.getSpaceByName("Small office 1")
+    open    = model.getSpaceByName("Open area 1")
+    utility = model.getSpaceByName("Utility 1")
+    plenum  = model.getSpaceByName("Level 0 Ceiling Plenum")
+    expect(entry).to_not be_empty
+    expect(office).to_not be_empty
+    expect(open).to_not be_empty
+    expect(utility).to_not be_empty
+    expect(plenum).to_not be_empty
+    entry   = entry.get
+    office  = office.get
+    open    = open.get
+    utility = utility.get
+    plenum  = plenum.get
+    expect(plenum.partofTotalFloorArea).to be false
+    expect(mod1.unconditioned?(plenum)).to be false
+
+    # TOTAL plenum roof area (4x surfaces), no overhangs.
+    roofs = mod1.facets(plenum, "Outdoors", "RoofCeiling")
+    total = roofs.sum(&:grossArea)
+    expect(total.round(2)).to eq(82.21)
+
+    gra_seb = mod1.grossRoofArea(model.getSpaces)
+    expect(mod1.status).to be_zero
+    expect(gra_seb.round(2)).to eq(total.round(2))
+
+    srr = 0.04
+
+    # The method returns the GRA, calculated BEFORE adding skylights/wells.
+    rm2 = mod1.addSkyLights(model.getSpaces, {srr: srr})
```

Comment:
Generates the following:

<img width="664" alt="seb_sky" src="https://github.com/rd2/osut/assets/98104985/b2e79e45-f60d-412b-b1af-efd34de1d5f8">